src/nodefony/bundles/http-bundle/src/context/websocket/websocket.js

```diff
@@ -100,8 +100,10 @@ nodefony.register.call(nodefony.context, "websocket", function () {
     connect() {
       return new Promise((resolve, reject) => {
         try {
+          let acceptedProtocol = null;
           if( this.resolver ){
-            this.connection = this.request.accept(this.resolver.acceptedProtocol || null, this.origin /*, this.request.cookies || null*/ );
+            acceptedProtocol = this.resolver.acceptedProtocol;
+            this.connection = this.request.accept(acceptedProtocol || null, this.origin /*, this.request.cookies || null*/ );
           }else{
             this.connection = this.request.accept( null, this.origin /*, this.request.cookies || null*/ );
           }
```

Comment:
```suggestion
          let acceptedProtocol = null;
          if( this.resolver ){
            acceptedProtocol = this.resolver.acceptedProtocol;
          }
          this.connection = this.request.accept(acceptedProtocol, this.origin /*, this.request.cookies || null*/ );
```
I don't get why you would need `|| null` or write twice the call